ddtrace/context.py

```diff
@@ -179,10 +163,10 @@ def get(self):
         This operation is thread-safe.
         """
         with self._lock:
+            sampled = any(span.sampled for span in self._trace)
```

Comment:
`get()` gets called after every single span finishes, so we probably want to guard this in the `if` statements.

Reply:
Ok, moved!